controllers/webhooks/workloads/integration/cfapp_validation_integration_test.go

```diff
@@ -93,7 +94,7 @@ var _ = Describe("CFAppValidatingWebhook", func() {
 			})
 
 			It("should fail", func() {
-				Expect(createErr).To(MatchError(ContainSubstring("CFApp with the same spec.name exists")))
+				Expect(createErr).To(MatchError(ContainSubstring(fmt.Sprintf("App with the name '%s' already exists.", app1Name))))
```

Comment:
You can remove the Sprintf call, since the matcher will do that for you. `ContainsSubstring("App with the name '%s' already exists.", app1Name)`